plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -592,6 +594,15 @@ private function add_feature_settings( $settings, $current_section ): array {
 		$features = $this->get_features( true );
 
 		$feature_ids              = array_keys( $features );
+		usort( $feature_ids, function( $feature_id_a, $feature_id_b ) use ( $features ) {
+			if ( ! isset( $features[ $feature_id_a ]['order'] ) ) {
+				$features[ $feature_id_a ]['order'] = 0;
+			}
+			if ( ! isset( $features[ $feature_id_b ]['order'] ) ) {
+				$features[ $feature_id_b ]['order'] = 0;
+			}
+			return $features[ $feature_id_b ]['order'] <=> $features[ $feature_id_a ]['order'];
```

Comment:
We can simplify the callback by using the `??` operator, as we wouldn't have to check if `'order'` is set in the `$features` entries nor set it ourselves.
Totally optional, though.

```suggestion
			return ( $features[ $feature_id_b ]['order'] ?? 0 ) <=> ( $features[ $feature_id_a ]['order'] ?? 0 );
```

Reply:
added in [a7614c7](https://github.com/woocommerce/woocommerce/pull/39912/commits/a7614c7e8f08017472ed3f4b21c00cf78b290f27) -- thanks a lot for the suggestion ☺️